app/src/main/java/org/mozilla/focus/components/RelocateService.java

```diff
@@ -56,14 +62,32 @@ public static void startActionMove(@NonNull Context context,
         intent.putExtra(Constants.EXTRA_DOWNLOAD_ID, downloadId);
         intent.putExtra(Constants.EXTRA_FILE_PATH, srcFile.getAbsolutePath());
         intent.setType(mediaType);
-        context.startService(intent);
+        ContextCompat.startForegroundService(context, intent);
+    }
+
+    private void startForeground() {
+        final NotificationCompat.Builder builder;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            builder = new NotificationCompat.Builder(getApplicationContext(), NotificationChannel.DEFAULT_CHANNEL_ID);
+        } else {
+            builder = new NotificationCompat.Builder(getApplicationContext());
```

Comment:
NotificationCompat.Builder(Context ...) is deprecated. Maybe it's fine just to use NotificationCompat.Builder(Context,String)} instead since we are already using compact version.